pkg/identities/service.go

```diff
@@ -163,9 +168,55 @@ func (s *Service) CreateIdentity(ctx context.Context, bodyID *kClient.CreateIden
 
 	s.authz.SetCreateIdentityEntitlements(ctx, identity.Id)
 
+	err = s.SendUserCreationEmail(ctx, identity)
+
 	return data, err
 }
 
+func (s *Service) SendUserCreationEmail(ctx context.Context, identity *kClient.Identity) error {
+	ctx, span := s.tracer.Start(ctx, "identities.Service.SendUserCreationEmail")
+	defer span.End()
+
+	template, err := mail.LoadTemplate(mail.UserCreationInvite)
+	if err != nil {
+		return err
+	}
+
+	code, link, err := s.generateRecoveryInfo(ctx, identity.Id)
+	if err != nil {
+		return err
+	}
+
+	emailAddress := identity.Traits.(map[string]interface{})["email"].(string)
+
+	userCreationInviteArgs := mail.UserCreationInviteArgs{
+		Email:        emailAddress,
+		InviteUrl:    link,
+		RecoveryCode: code,
+	}
+
+	err = s.email.Send(ctx, emailAddress, userCreationEmailSubject, template, userCreationInviteArgs)
```

Comment:
here we should chuck a context.WithTimeout and deal with the failure gracefully

Reply:
Right now I'm leveraging whatever timeout/deadline the request context has.
Should we add this in this PR or can we consider it an enhancement for another time?

Reply:
server timeout is probably 60s so it would be a problem 

add an issue in the repo and we ll tackle it next iteration 

fix should be quick nonetheless